CHANGES.rst

```diff
@@ -8,6 +8,43 @@ https://hyperspy.readthedocs.io/en/latest/user_guide/changes.html
 
 .. towncrier release notes start
 
+Hyperspy 1.6.3 (2021-06-10)
+===========================
+
+Bug Fixes
+---------
+
+- Fix ROI snapping regression (`#2720 <https://github.com/hyperspy/hyperspy/issues/2720>`_)
+- Fix :py:meth:`~._signals.signal1d.Signal1D.shift1D` regression with navigation dimension larger than one (`#2729 <https://github.com/hyperspy/hyperspy/issues/2729>`_)
```

Comment:
Maybe it'll be good to mention that this fixes the issue with ``align_zero_loss_peak``?